app/views/unattended/provisioning_templates/snippet/efibootmgr_netboot.erb

```diff
@@ -4,7 +4,9 @@ name: efibootmgr_netboot
 model: ProvisioningTemplate
 description: Configure booting from network in EFI
 snippet: true
-%>
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion?

Reply:
Most EFI system firmware implementations put new items to the top of the boot priority list. Foreman workflow assumes that managed nodes always boot from network. This snippet can be included from provisioning templates and when "efi_bootentry" host param is set to "previous" it will attempt to find previous boot entry (network) and put it back into first position.